lib/rubocop/cop/rspec/described_class.rb

```diff
@@ -102,11 +106,7 @@ def scope_change?(node)
         end
 
         def skippable_block?(node)
-          node.block_type? && !rspec_block?(node) && skip_blocks?
-        end
-
-        def skip_blocks?
-          cop_config['SkipBlocks']
+          node.block_type? && !rspec_block?(node) && cop_config['SkipBlocks']
```

Comment:
This change also looks like a ClassLength “fix”, but it ends up making the code _less_ clear, in my opinion.